build.sbt

```
@@ -105,20 +105,14 @@ lazy val sharedSettings = List(
   scalacOptions ++= crossSetting(
     scalaVersion.value,
     if2 = List("-Yrangepos", "-deprecation"),
-    if213 = List("-release", "8"),
-    if212 = List("-Xexperimental", "-target:jvm-1.8"),
+    if213 = List("-release", "11"),
+    if212 = List("-Xexperimental", "-target:11"),
```

```suggestion
    if212 = List("-Xexperimental", "-release", "8"),
```
we can only use the release flag I guess, but that should be fine since JDK 11 is anyway used for the release job.

---

👍 thanks